### DISCUSSION TITLE
https://greendolphindance.com/2024/02/05/Mac%E8%BF%90%E8%A1%8C%E4%BB%85%E6%94%AF%E6%8C%81Windows%E7%9A%84Steam%E6%B8%B8%E6%88%8F/

### DISCUSSION THREAD

**Discussion by giscus** [greendolphindance/greendolphindance.github.io#20] [General]
# https://greendolphindance.com/2024/02/05/Mac%E8%BF%90%E8%A1%8C%E4%BB%85%E6%94%AF%E6%8C%81Windows%E7%9A%84Steam%E6%B8%B8%E6%88%8F/

Back to the rough ground!

https://greendolphindance.com/2024/02/05/Mac%E8%BF%90%E8%A1%8C%E4%BB%85%E6%94%AF%E6%8C%81Windows%E7%9A%84Steam%E6%B8%B8%E6%88%8F/

**Comment by wang7peng** [comment]
最近一直在 windows 上用 `VMware17` 搭 mac 小窗口，本来不能这么干，下载 unlocker 破解改vmx文件一堆事，重启一次还要老半天，烦死了 :cry:。团队没有macOS，老板买了个 15年的旧的，结果发现装 sonoma 起码 2018 年之后生产的，好一点的二手都要 1w+，哎 🥺。

**Reply by wang7peng** [reply]
至于为什么二手也要这么好的 mac，因为现在 iphone 系统是 `ios 17.4`，这个模拟器必须从 app store 下最新的 `xcode` 才有，然后 `xcode` 要求系统必须是 `macOS13.5` 以上，然后装整个系统又现在了硬件。好嘛环环相扣，旧一点的还有 CPU 型号限制。然后现在天天搞 `VMware` 了，有时运行一下要半天，也不知道是卡了还是死了🙃，焦头烂额。

**Reply by greendolphindance** [reply]
那么为什么不猪里脊用mac呢！

**Reply by greendolphindance** [reply]
主力机…….什么输入法，馋了

**Comment by wang7peng** [comment]
因为没有 mac 啊，打算下个月买一台二手。看又卡机了😪
![](https://img2.imgtp.com/2024/03/14/M0562QJM.png)

累了就来看看大妹妹又更新了啥~哈哈哈
